gratipay/models/package/__init__.py

```diff
@@ -40,3 +51,35 @@ def from_names(cls, package_manager, name):
         """
         return cls.db.one("SELECT packages.*::packages FROM packages "
                           "WHERE package_manager=%s and name=%s", (package_manager, name))
+
+
+    @property
+    def team(self):
+        """A computed attribute, the :py:class:`~gratipay.models.team.Team`
+        linked to this package if there is one, otherwise ``None``. Makes a
+        database call.
+        """
+        return self._load_team(self.db)
+
+
+    def ensure_team(self, cursor, owner):
+        """Given a db cursor and :py:class:`Participant`, insert into ``teams`` if need be.
+        """
+        team = self._load_team(cursor)
```

Comment:
> I'd also like to look at why the second `finish_email_verification` call appears to succeed. I thought it was supposed to fail if the nonce is the same, even when called serially.

I do find [this test](https://github.com/gratipay/gratipay.com/blob/2cc9cd42dc11caeb01d40675e32b2e0b83cb0c1d/tests/py/test_email.py#L151-L157). The second call supposedly returns a "redundant" error code.